lib/whitespace.coffee

```diff
@@ -1,4 +1,5 @@
 {CompositeDisposable} = require 'atom'
+_ = require 'underscore'
```

Comment:
You might want to use `underscore-plus` instead since it is already a dependency in many Atom packages.